tests/playwright/src/model/pages/preferences-page.ts

```diff
@@ -19,20 +19,71 @@
 import type { Locator, Page } from '@playwright/test';
 import test, { expect as playExpect } from '@playwright/test';
 
+import { delay } from '/@/utility/wait';
+
 import { SettingsPage } from './settings-page';
 
 export class PreferencesPage extends SettingsPage {
   readonly heading: Locator;
+  readonly searchbar: Locator;
   readonly kubePathInput: Locator;
 
   constructor(page: Page) {
     super(page, 'Preferences');
     this.heading = this.header.getByLabel('Title', { exact: true });
+    this.searchbar = this.header.getByLabel('search preferences');
```

Comment:
why not to use the search to narrow down the particular sub-section of each preferences configuration?

Reply:
@cbr7 you're correct that it's not used, however this is a page fragment model, does it really hurt to have the search bar defined? other extensions can use the element in their code too. I don't see a downside here.

Reply:
@odockal I've implemented opening specific subsection of the preferences, doing an additional search action wouldn't help anything at this point.